azurelinuxagent/ga/exthandlers.py

```diff
@@ -191,21 +191,28 @@ def __init__(self):
         self.log_process = False
 
         self.report_status_error_state = ErrorState()
+        self.get_artifact_error_state = ErrorState(min_timedelta=datetime.timedelta(minutes=5))
```

Comment:
5 should be a constant or the default for ErrorState.

Reply:
Sure, will update